test/ectest.c

```diff
@@ -1430,7 +1378,7 @@ static int nistp_single_test(int idx)
         || !TEST_int_eq(0, EC_POINT_cmp(NISTP, Q, G, ctx)))
       goto err;
 
-    r = group_order_tests(NISTP);
+    r = 1;
```

Comment:
We are removing this, because it is effectively already tests in `prime_field_tests()` and this now doesn't add any value anymore, right?

Reply:
Yes exactly